fcrepo-http-api/src/main/java/org/fcrepo/http/api/services/HttpRdfService.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.http.api.services;
+
+import static org.apache.jena.rdf.model.ModelFactory.createDefaultModel;
+import static org.fcrepo.kernel.api.rdf.DefaultRdfStream.fromModel;
+import static org.apache.jena.riot.RDFLanguages.contentTypeToLang;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.core.MediaType;
+import org.apache.jena.atlas.RuntimeIOException;
+import org.apache.jena.rdf.model.impl.StatementImpl;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RiotException;
+import org.fcrepo.http.commons.api.rdf.HttpIdentifierConverter;
+import org.fcrepo.kernel.api.exception.MalformedRdfException;
+import org.fcrepo.kernel.api.exception.RepositoryRuntimeException;
+import org.fcrepo.kernel.api.RdfStream;
+import org.slf4j.Logger;
+
+/**
+ * A service that will translate the resourceURI to Fedora ID in the Rdf InputStream
+ *
+ * @author bseeger
+ * @author bbpennel
+ * @since 2019-11-07
+ */
+
+public class HttpRdfService {
+
+    private static final Logger log = getLogger(HttpRdfService.class);
+
+    @Inject
+    private HttpIdentifierConverter idTranslator;
+
+    /**
+     * Parse the request body to a Model, with the URI to Fedora ID translations done.
+     *
+     * @param resourceId the ID of the Fedora resource
```

Comment:
might want to change the variable name to reflect that it must be an external uri representation of the resource identifier in all these methods, or at least the javadoc